curriculum/js1/8.js

```diff
@@ -6,7 +6,24 @@
  */
 
 const solution = (a, b) => {
```

Comment:
I don't think this is the right approach. Feel free to change the function shape `solution = (a, b, i=a) => {...`
decrease i by 1 untill its divisible by both a and b